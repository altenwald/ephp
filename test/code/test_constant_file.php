<?php

var_dump(constant("__FILE__"));
