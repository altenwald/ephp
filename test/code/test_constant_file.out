
Warning: constant(): Couldn't find constant __FILE__ in {{CWD}}/test/code/test_constant_file.php on line 3
NULL
